job-dsl-core/src/main/groovy/javaposse/jobdsl/dsl/helpers/publisher/PublisherContext.groovy

```diff
@@ -1579,6 +1579,26 @@ class PublisherContext extends AbstractExtensibleContext {
     }
 
     /**
+     * Sends build status and coverage information to Pharbicator
+     *
+     * @since 1.39
+     */
+    @RequiresPlugin(id = 'phabricator-plugin', minimumVersion = '1.5')
```

Comment:
Hmm weird https://wiki.jenkins-ci.org/display/JENKINS/Phabricator+Plugin "Latest Release" shows 1.5 even though the last published one is 1.8.1.